src/TimeZones.jl

```diff
@@ -1,5 +1,7 @@
 module TimeZones
 
+using RelocatableFolders
```

Comment:
Please add this after `using RecipesBase`. Can we get away with `using RecipesBase: @path`?